### DISCUSSION TITLE
Is the bridge live on Kusama?

### DISCUSSION THREAD

**Discussion by jommi9** [Snowfork/snowbridge#830] [General] [answered]
I saw that you had a milestone of launching the bridge on Kusama by April - did that happen?

**Comment by vgeddes** [comment, marked as answer]
Hey @jommi9,

We haven't launched on Kusama yet. Unfortunately there have been delays.

Particularly, we have had to re-architecture our bridge to operate on the BridgeHub system parachain, and to use XCMv3 as the underlying messaging protocol. The original bridge was designed in ways that were incompatible with Polkadot's system parachain architecture.

The scope for this work wasn't really clearly defined when we submitted our treasury proposal. Much of the design work was only completed afterwards. The re-architecture ended up taking longer than expected.

Another delay has been the recent understanding that a core part of our bridge security - the Ethereum ALC procotocol - has some [security flaws](https://forum.polkadot.network/t/snowforks-analysis-of-sync-committee-security/2712). Analysing the situation and devising mitigations has been costly from a resource perspective, also contributing to the delay.

I think its important to note that even if we had everything ready our side, we still depend on several pending deliverables from Parity:
1. Audits for BEEFY, and activation of BEEFY on Kusama
2. Upgrading BridgeHub system parachain to support bridges (currently BridgeHub is little more than a shell parachain) 
3. Upgrading Statemine to support asset transfers to external chains (i.e transfer to Ethereum, Polkadot)

So this bridge project is a collaboration with Parity, and we're working together daily to bring the project to fruition.

We mentioned all these things as risk factors in the "Launch Plan" section of our proposal.

**Reply by jommi9** [reply]
Thank you for the indepth reply!

**Comment by vgeddes** [comment]
Should be deployed by end of Q1 2024.

**Reply by vgeddes** [reply]
See latest update here: https://forum.polkadot.network/t/update-on-snowbridge/3902/5?u=vincent
